docs/content/en/docs/Getting started/_index.md

```diff
@@ -14,38 +14,89 @@ The same steps works on a barametal host, however, your mileage and configuratio
 
 ## Prerequisites
 
-- A VM hypervisor that boots ISOs ( or, a baremetal )
+
+{{% alert title="Warning" %}}
+Keep in mind the size of the resulting image. This will vary between distributions but you should be safe if you have 30+ Gb available disk space.
```

Comment:
I think this is a bit misleading, which image? Maybe we should just mention disk size as requirement

Reply:
good point, removed the alert all together